src/cli/main.py

```diff
@@ -212,8 +214,8 @@ def get_job(ctx: click.Context, name: str) -> None:
 @click.option("--template-name", "-t", type=str, help="Job template name.")
 @click.option("--name", "-n", help="Job template name.", required=True)
 @click.option("--vm-size", "-s", help="VM size.")
-@click.option("--priority", "-p", help="A comma separated list of LISA test priority to run.")
-@click.option("--case-name", "-c", help="A comma separated list of LISA test case to run.")
+@click.option("--priority", "-p", "test_priorities", multiple=True, help="Test priority to run.")
+@click.option("--case-name", "-c", "test_cases", multiple=True, help="Test case to run.")
```

Comment:
`Test cases to run` (plural)

Reply:
Really it depends how you read it. It will look like this
```
aitl-cli --case-name xxx --case-name yyy --case-name zzz
```
So it should be singular IMO

Reply:
What does the help output look like though?

Reply:
Is it something like `--case-name <first_case> [--case-name <second_case>] [--case-name <third_case>]...`? If yes, then I believe `"Test cases"` would be appropriate.

Reply:
Thanks for clarifying. I agree that singular is appropriate.